products/pluto/pluto_build/sql/export_ae_tables.sql

```diff
@@ -0,0 +1,97 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+DROP TABLE IF EXISTS ae_tax_lot;
+CREATE TABLE ae_tax_lot AS
+SELECT
+    e.bbl::CHAR(10),
+    e.borocode::CHAR(1) AS borough_id,
+    e.block::TEXT,
+    e.lot::TEXT,
+    e.address::TEXT,
+    e.landuse::CHAR(2) AS land_use_id,
+    ST_TRANSFORM(g.geom_2263, 4326) AS wgs84,
+    g.geom_2263 AS li_ft
+FROM export_pluto AS e
+INNER JOIN pluto_geom AS g ON e.bbl = g.bbl::NUMERIC
+WHERE g.geom_2263 IS NOT NULL;
+
+DROP TABLE IF EXISTS ae_zoning_district;
+CREATE TABLE ae_zoning_district AS
+SELECT
+    GEN_RANDOM_UUID() AS id,
+    zonedist AS label,
+    geom AS wgs84,
+    ST_TRANSFORM(geom, 2263) AS li_ft
+FROM dcp_zoningdistricts
+WHERE
+    zonedist NOT IN ('PARK', 'BALL FIELD', 'PUBLIC PLACE', 'PLAYGROUND', 'BPC', '')
+    AND ST_GEOMETRYTYPE(ST_MAKEVALID(geom)) = 'ST_MultiPolygon';
+
+
+DROP TABLE IF EXISTS ae_zoning_district_zoning_district_class;
+CREATE TABLE ae_zoning_district_zoning_district_class AS
+WITH split_zones AS (
+    SELECT
+        id,
+        UNNEST(STRING_TO_ARRAY(label, '/')) AS individual_zoning_district
+    FROM ae_zoning_district
+)
+SELECT
+    id AS zoning_district_id,
+    (REGEXP_MATCH(individual_zoning_district, '^(\w\d+)(?:[^\d].*)?$'))[1] AS zoning_district_class_id
+FROM split_zones;
+
+DROP TABLE IF EXISTS ae_zoning_district_class_colors;
+CREATE TABLE ae_zoning_district_class_colors AS
+SELECT t.* FROM (
```

Comment:
Interesting... I'd (naively) think that AE would want to control the colors for these on the front end

Reply:
I agree. And also in general I'd argue it's not something that philosophically belongs here. But for now this works